content/docs/pipelines/deployment-alternatives.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Pipelines Standalone Deployment Alternatives"
```

Comment:
is "alternatives" necessary?

Reply:
Placeholder for introducing marketplace later

Reply:
Thanks for updating the terminology from "lite/lightweight" to "standalone" and for ensuring consistency of this terminology!

Now that we have "standalone" in the title, let's remove "alternatives". We can add it back later if necessary, but it's clearer to keep future possibilities out of the docs until they become a reality. Otherwise people become confused.

Reply:
will remove alternatives